ploop.go

```diff
@@ -191,6 +191,10 @@ func UmountByDevice(dev string) error {
 	return ploop("umount", "-d", dev)
 }
 
+func UmountByMount(mountpoint string) error {
```

Comment:
Please add the comment before the function with a brief description. This is used by godoc.

After the comment is added, check it with golint (or go vet -- I don't remember).

Reply:
Also, I would rename it to `UmountByMountPoint` to avoid confusion.